Source/SG/SG.cs

```diff
@@ -23,19 +23,39 @@ public struct InstrumentConfiguration
             public double CarrierFrequency_Hz;
             public double DutAverageInputPower_dBm;
             public double ExternalAttenuation_dBm;
-            public bool ShareLOSGToSA;
+            public LocalOscillatorConfiguration[] LoConfigurations;
+            
             public static InstrumentConfiguration GetDefault()
             {
                 return new InstrumentConfiguration()
                 {
-                    SelectedPorts="",
+                    SelectedPorts = "",
                     ReferenceClockSource = RfsgFrequencyReferenceSource.PxiClock.ToString(),
                     CarrierFrequency_Hz = 1e9,
                     DutAverageInputPower_dBm = 0,
                     ExternalAttenuation_dBm = 0,
-                    ShareLOSGToSA = true,
+                    LoConfigurations = new LocalOscillatorConfiguration[] { LocalOscillatorConfiguration.GetDefault() }
                 };
             }
+
+            public static InstrumentConfiguration GetDefault(NIRfsg sessionHandle)
+            {
+                InstrumentConfiguration instrConfig = new InstrumentConfiguration();

```

Comment:
Same feedback as line 22 in Instr module:
```suggestion
                InstrumentConfiguration instrConfig = InstrumentConfiguration.GetDefault();
```